ci-operator/jobs/openshift/oadp-operator/openshift-oadp-operator-master-ocp-4.10-presubmits.yaml

```diff
@@ -0,0 +1,399 @@
+presubmits:
+  openshift/oadp-operator:
+  - agent: kubernetes
+    always_run: false
+    branches:
```

Comment:
Can we instead of having a file for each branch, add all branches needed for each ocp version file? (the periodics syntax is different, maybe just for pre and post submits)

Reply:
i closed this PR, it's wrong way to go as `make jobs` is generating the jobs, so we should not touch them